README.md

```diff
@@ -1 +1,85 @@
-# book-dirt-bike-front-end
\ No newline at end of file
+# Dirt Bikes
+
+> Dirt Bikes is an application to book dirt bikes for rent. The user will be able to choose a bike according to his preferences and book a period during which he will use the desired bike.
+
+<h2 align="center">
+  Demo Pic📝
+</h2>
+
+![image](/src/components/images/WhatsApp%20Image%202022-09-29%20at%2014.17.39.jpeg)
+
+
+## Built With
+
+- Create-react-app
+- Javascript
+- CSS
+- JSX
+- React router
+- Redux
+
+## Back-end repo
+[backend repo](https://github.com/Tufahel/book-dirt-bike-back-end)
+
+**Live**
+
+- Live app: [Dirt-Bike](https://incredible-unicorn-5f0c7d.netlify.app/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these setup steps.
+
+**Setup**
+
+- Open terminal and clone the project: `git clone git@github.com:Tufahel/book-dirt-bike-front-end.git`
+- cd into the directory `book-dirt-bike-front-end`
+
+**Prerequisites**
+
+- You need to have `node.js` installed in your system
+
+**Install**
+
+- On your terminal run : `npm i`
+- To run the app in the development mode : `npm start`
+- Launch the test runner : `npm test`
+- In case you run into any errors please raise an [issue](https://github.com/Tufahel/book-dirt-bike-front-end/issues).
+
+## Authors
+
+👤 **Toyosi Taiwo**
+
+- GitHub: [@Yorsyboy](https://github.com/Yorsyboy)
+- Twitter: [@tboytaiwo](https://twitter.com/Tboytaiwo)
+- LinkedIn: [Toyosi Taiwo](https://linkedin.com/in/taiwo-toyosi)
+
+👤 **Tufahel Ahmed**
+
+- GitHub: [@Tufahel](https://github.com/Tufahel)
+- Twitter: [@TufahelAhmed](https://twitter.com/TufahelAhmed)
+- LinkedIn: [@Tufahel-Ahmed](https://www.linkedin.com/in/tufahel-ahmed/)
+
+👤 **Nii Amanor Djoleto**
+
+- Github: [@Ramrod433](https://github.com/ramrod433)
+- Twitter: [@_Djoleto](https://twitter.com/_djoleto_)
+- LinkedIn: [Nii Djoleto](https://www.linkedin.com/in/nii-amanor-djoleto)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Credits
+
+Design inspiration: [Murat Korkmaz on Behance](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
- [ ] Your project lacks a license file and a license section on your readme.  Please, kindly add the license file and corresponding readme section with a link to it. 

You can go [here]https://github.com/microverseinc/readme-template and grab the MIT.md file, put it at the root of your project, and link to it on your readme.

Reply:
Thanks for your your review @Meltrust. We have fixed it.